tests/Domain/Value/BranchTest.php

```diff
@@ -27,7 +27,7 @@ final class BranchTest extends TestCase
   variants:
     symfony/symfony: ['4.4']
     sonata-project/block-bundle: ['4']
-  services: []
```

Comment:
php_extensions:

Is missing

Reply:
I will open next PR for this sugestion in few hours.

Reply:
Thank you 🚀

Reply:
I just came to my office. I see I dont have to make PR for it. Thanks @VincentLanglet. PR is reviewed.